python/models.md

```diff
@@ -0,0 +1,9 @@
+# Model style
+
+```
+title = models.CharField(
+    max_length=100,
+    blank=True,
+    null=True,
```

Comment:
Yea, i understand the reasoning and i do like it for that reason. I'm easy either way tbf i think we should just be consistent across languages.

So i'd advise adding a dangling comma to our ESLint